.redhat-ci.yml

```diff
@@ -0,0 +1,25 @@
+branches:
+    - master
+    - auto
+    - try
+
+container:
+    image: jlebon/ostree-tester:rhci
```

Comment:
Right, this will definitely need to be set up as an automated build. Any preferences on which org? (Or I don't mind leaving it under my name either).